src/control/fault/fault.go

```diff
@@ -112,7 +112,7 @@ func (f *Fault) Equals(raw error) bool {
 	if !ok {
 		return false
 	}
-	return f.Code == other.Code
+	return f.Code == other.Code && f.Description == other.Description
```

Comment:
Just wondering about the motivation for this change. Shouldn't all faults with the same code also have the same description?

Reply:
currently you can create faults with the same code but different descriptions, @mjmac asked me to add this.

Reply:
Yeah, I suggested this as a "belt and suspenders" approach. We shouldn't reuse codes, but I think I'm guilty of adding a few non-unique faults. Will fix those up later.